strawberry/schema/schema_converter.py

```diff
@@ -126,12 +128,14 @@ def from_directive(self, directive: DirectiveDefinition) -> GraphQLDirective:
     def from_field(self, field: StrawberryField) -> GraphQLField:
         field_type: GraphQLType
 
-        if isinstance(field.type, StrawberryOptional):
-            field_type = self.from_optional(field.type)
+        resolved_type = field.resolved_type
```

Comment:
do we need this variable seems redundant :D

Reply:
It's just so I don't have to access it multiple times